src/fonts/lib/Space.ttx

```diff
@@ -236,7 +184,7 @@ http://scripts.sil.org/OFL
       <PaintType value="0"/>
       <CharstringType value="2"/>
       <FontMatrix value="0.001 0 0 0.001 0 0"/>
-      <FontBBox value="0 0 300 533"/>
+      <FontBBox value="50 0 200 533"/>
       <StrokeWidth value="0"/>
```

Comment:
This changed because of the removal of `0xeffe` and `0xefff`.  How did you determine the new values?